src/features/MentorPage/components/MentorList/MentorCard/List/Header.tsx

```diff
@@ -48,27 +49,41 @@ export const Header: React.FC<Props> = ({
   };
 
   return (
-    <Container isAvailable={isAvailable} isMobile={isMobile}>
+    <Container isAvailable={isAvailable} isMobile={isMobile} isMe={isMe}>
       <Tag status={getStatus(isMe, isAvailable, isNew)}></Tag>
-      <ProfilePicture />
+      <ProfilePicture isMe={isMe} />
       <BasicInfo>
-        <NameText variant="h2" color="white">
+        <NameText variant="h2" color={isMe ? 'blueDark' : 'white'}>
           {name}
         </NameText>
-        <WrappedText color="white">
+        <WrappedText color={isMe ? 'blueDark' : 'white'}>
           {age} {t('card.age')} <Divider>|</Divider>
           {region}
         </WrappedText>
-        <TruncateText color="white">{message}</TruncateText>
+        <TruncateText color={isMe ? 'blueDark' : 'white'}>
+          {message}
+        </TruncateText>
       </BasicInfo>
     </Container>
   );
 };
 
-const Container = styled.div<{ isAvailable: boolean; isMobile: boolean }>`
+const Container = styled.div<{
+  isAvailable: boolean;
+  isMobile: boolean;
+  isMe: boolean;
+}>`
   align-items: center;
-  background-color: ${({ isAvailable }) =>
-    isAvailable ? palette.purple : palette.blueGrey};
+  background-color: ${({ isAvailable, isMe }) => {
+    if (!isAvailable) {
+      return palette.blueGrey;
+    }
+    if (isMe) {
+      return palette.blue;
+    } else {
+      return palette.purple;
+    }
+  }};
```

Comment:
I would suggest the same statusColors-map refactor as I did in my other comment!